modules/bot/Module.php

```diff
@@ -150,20 +175,52 @@ private function initialize($update, $botId)
                 'first_name' => $updateChat->getFirstName(),
                 'last_name' => $updateChat->getLastName(),
             ]);
+
             if (!$telegramChat->save()) {
                 return false;
             }
 
+            // Add chat administrators to db
+            if ($newChat && $updateChat->getType() != Chat::TYPE_PRIVATE) {
+                $administrators = $this->botApi->getChatAdministrators($updateChat->getId());
+
+                foreach ($administrators as $administrator) {
+                    $chatMember = new ChatMember();
+
+                    $chatMember->setAttributes([
+                        'chat_id' => $telegramChat->id,
+                        'telegram_user_id' => $administrator->getUser()->getId(),
+                        'status' => $administrator->getStatus(),
+                    ]);
+
+                    $chatMember->save();
+                }
+            }
+
             // To separate commands for each type of chat
             $namespace = $telegramChat->isPrivate()
                 ? Controller::TYPE_PRIVATE
                 : Controller::TYPE_PUBLIC;
             $this->setupPaths($namespace);
 
-            if (!in_array($telegramUser, $telegramChat->getUsers()->all())) {
-                $telegramChat->link('users', $telegramUser);
+            $chatMember = ChatMember::find()->where(['chat_id' => $telegramChat->id, 'telegram_user_id' => $telegramUser->provider_user_id])->one();
+
+            if (!isset($chatMember)) {
+                $chatMember = new ChatMember();
```

Comment:
The same as in [this comment](https://github.com/opensourcewebsite-org/opensourcewebsite-org/pull/286/files#r388152940)